apps/next/db/schema.ts

```diff
@@ -267,9 +267,10 @@ export const companyRoleRates = pgTable(
     updatedAt: timestamp("updated_at", { precision: 6, mode: "date" })
       .notNull()
       .$onUpdate(() => new Date()),
-    payRateInSubunits: integer("pay_rate_in_subunits").notNull(),
+    payRateInSubunits: integer("pay_rate_in_subunits"),
     payRateCurrency: varchar("pay_rate_currency").default("usd").notNull(),
     trialPayRateInSubunits: integer("trial_pay_rate_in_subunits").notNull(),
```

Comment:
This seems to have been missed.

```suggestion
    trialPayRateInSubunits: integer("trial_pay_rate_in_subunits"),
```